tests/example_data.ts

```diff
@@ -3,6 +3,57 @@
 import * as fs from "fs";
 import * as path from "path";
 
+export const patient_expectations = {
```

Comment:
```suggestion
export const patientExpectations = {
```

Reply:
Rest of the file should be camelCased variables too as that's idiomatic in JS/TS.

Reply:
Thanks. I'd meant to switch that!